packages/@react-spectrum/color/docs/ColorField.mdx

```diff
@@ -0,0 +1,180 @@
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<ColorField label="Primary Color" />
+```
+
+## Value
+
+A ColorField's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop. Valid values are hexadecimal color codes.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState('#e73623');
+
+  return (
+    <Flex gap="size-150" wrap>
+      <ColorField
+        label="Primary Color (Uncontrolled)"
+        defaultValue="#e21" />
+
+      <ColorField
+        label="Primary Color (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Placeholder text, provided using the `placeholder` prop, describes the expected value or formatting for the ColorField.
+Placeholder text will only appear when the ColorField is empty, and should not be used as a substitute for labeling the
+component with a visible label.
+
+<ColorField placeholder="#e73623" label="Primary Color" />
+
+## Labeling
+
+A visual label should be provided for the ColorField using the `label` prop. If the ColorField is required, the `isRequired` and
+`necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<Flex gap="size-150" wrap>
+  <ColorField label="Primary Color" />
+  <ColorField label="Primary Color" isRequired />
+  <ColorField label="Primary Color" isRequired necessityIndicator="label" />
+  <ColorField label="Primary Color" necessityIndicator="label" />
+</Flex>
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the ColorField for
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using
+the `id` of the labeling element instead.
+
+### Internationalization
+
+In order to internationalize a ColorField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+## Step values
+
+The `step` prop can be used to snap the value to certain increments. The steps are calculated starting from
+zero or the current value. A `step` can be set to any positive decimal, defaulting to `1` if unspecified.
+
+```tsx example
+<ColorField label="Primary Color" step={16} />
+```
+
+## Validation
+
+ColorField can display a validation state to communicate to the user whether the current value is valid or invalid.
+Implement your own validation logic in your app and pass either `"valid"` or `"invalid"` to the ColorField via the `validationState` prop.
+
+The example below illustrates how one would validate if the user has entered a red color.
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState('#e73623');
```

Comment:
If I use the stately color package like Daniel did https://github.com/adobe/react-spectrum/pull/1632/files#diff-addf8a33459e32a12f6d20f3f399163936469198e52834f7413162debace95d2R57, I could get rid of the if statement, simplifying the example. I choose to not do that because I wasn't sure if people reading this example and creating an app using React Spectrum would do that too. The context of what Daniel is showing is slightly different, but it still makes me unsure what is the best choice here.

Reply:
Yes we want people to take advantage of the parse function. I would add a popover link in the value section or something? Something like this https://reactspectrum.blob.core.windows.net/reactspectrum/d5112eb2e9b59888ec5217353bf1fd83566a6c03/docs/react-aria/useColorWheel.html#controlled